File: 3174-clear-digits/3174-clear-digits.cpp
class Solution {
public:
    string clearDigits(string s) {
        string ans = "";
        for (int i = 0; i < s.size(); i++) {
            if (isdigit(s[i])) {
                ans.pop_back();
            } else ans += s[i];
        }
        return ans;
    }
};

File: 3174-clear-digits/3174-clear-digits.go
func clearDigits(s string) string {
	st := []rune{}

	for _, v := range s {
		if '0' <= v && v <= '9' {
			st = st[:len(st) - 1]
			continue
		}
		st = append(st, v)
	}

	return string(st)
}